hack/ci/devstack-cloud-init.yaml.tpl

```diff
@@ -78,8 +80,22 @@ write_files:
 
       # Don't download default images, just our test images
       DOWNLOAD_DEFAULT_IMAGES=False
-      IMAGE_URLS="https://github.com/sbueringer/image-builder/releases/download/v1.18.15-01/ubuntu-2004-kube-v1.18.15.qcow2,"
-      IMAGE_URLS+="http://download.cirros-cloud.net/0.5.1/cirros-0.5.1-x86_64-disk.img"
+      # We upload the Amphora image so it doesn't have to be build
```

Comment:
maybe a TODO here

Reply:
I would keep it for now. This adds a dependency but we already have the dependency because of the ubuntu image but this issue will try to fix it: https://github.com/kubernetes-sigs/cluster-api-provider-openstack/issues/794

(it just takes a long time to build the amphora image during the devstack installation on each test)